src/loader.ts

```diff
@@ -385,6 +389,40 @@ interface ModuleDefinitionArguments extends Array<any> {
 		pendingCacheInsert = {};
 	}
 
+	function loadNodeModule(moduleId: string, parent?: Module): any {
```

Comment:
it was out of scope when the new global stuff was introduced, couldn't fully determine why, but I believe it was garbage collected maybe?